NitroxClient/Debuggers/SceneDebugger.cs

```diff
@@ -237,31 +237,43 @@ private void RenderTabScenes()
             using (new GUILayout.VerticalScope("Box"))
             {
                 GUILayout.Label("All scenes", "header");
-                for (int i = 0; i < SceneManager.sceneCountInBuildSettings; i++)
+                for (int i = 0; i < SceneManager.sceneCountInBuildSettings+1; i++)
                 {
-                    Scene currentScene = SceneManager.GetSceneByBuildIndex(i);
-                    string path = SceneUtility.GetScenePathByBuildIndex(i);
+                    Scene currentScene;
+                    string path = "";
+                    if (i == SceneManager.sceneCountInBuildSettings)
+                    {
+                        currentScene = NitroxBootstrapper.Instance.gameObject.scene; // bit of a hack
```

Comment:
Could this be done instead with

 - https://docs.unity3d.com/ScriptReference/SceneManagement.Scene-name.html
OR
 - https://docs.unity3d.com/540/Documentation/ScriptReference/SceneManagement.SceneManager.GetActiveScene.html

Reply:
I used the i == SceneManager.sceneCountInBuildSettings instead of string equality for some performance reasons, also GetActiveScene will likely return the main scene and will not include the Scenes that are loaded as Additive, SceneManager.sceneCountInBuildSettings is always going to be the same number in a release build so simply adding 1 to it and then checking if the indexer is equal to that number is acceptable, I would've just simply moved it outside of the loop however that would require moving the Using further up the method(out side of the for loop) and duplicating logic